test/models/api_resource_test.rb

```diff
@@ -1,7 +1,39 @@
 require "test_helper"
 
 class ApiResourceTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "when initialized - inherits parent properties" do
```

Comment:
@Pralish we are waiting on you to author [these tests](https://github.com/restarone/violet_rails/pull/558/files#diff-8449e3abaeac32d30d2730783c534a9e29aed06dbc252c491093ee4b0b27d8fdR4-R38) before we merge to `master`!